pages/docs/support/rest-api.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: "Grouparoo REST API"
+date: "2021-08-02"
+pullQuote: "Use the Grouparoo REST API to connect to your Grouparoo instance from an external application."
+---
+
+Grouparoo's REST API enables you to connect to your Grouparoo instance from an external application and return data in JSON format.
+
+## What can you do with our API?
+
+Grouparoo's REST API contains endpoints for over 25 topics -- everything from installing a Grouparoo Plugin to getting a list of all your Schedules. To see a full list of API endpoints, navigate to `/swagger` in the UI of your Grouparoo instance or run:
+
+```bash
+curl -X 'GET' \
+  '{{yourinstanceurl}}/api/v1/swagger' \
+  -H 'accept: application/json'
+```
+
+with your instance's URL.
+
+## Using the Grouparoo API
+
+### Generating an API Key
+
+To get started using Grouparoo's REST API, you'll need to generate an API key. To do this, first make sure you are signed into an account with "write" access for API Keys. Then:
+
+#### UI Method
+
+1. Go to Platform > API Keys or navigate to `/apiKeys` and select "Add API Key".
+
+2. Give your API Key a name. An API Key will be generated and you will be prompted to set permissions for this API.
+
+<ImageInBrowserFrame
+  centered
+  source="Generate an API Key"
+  alt="Generate an API Key"
+  src="/images/docs/support/rest-api/apikeyconfig.png"
+  width={820}
+  height={502}
+/>
+
+To learn more about what each of the topics can do in the API, you can access the Swagger Docs for the Grouparoo API by navigating to `/swagger` in the UI of your Grouparoo Instance or going to Platform > API Documentation.
+
+#### Code Config Method
+
+You can also generate an API key object with the [`grouparoo generate` command]. To do this, run
+
+```
+grouparoo generate apiKey {{yourAPIKeyId}}
+```
+
+This command will generate a config file in a new directory `/config/apiKeys/{{yourAPIKeyId}}.js` that looks like this:
+
+```js
+exports.default = async function buildConfig() {
+  return [
+    {
+      class: "apiKey",
+      id: "{{yourAPIKeyId}}",
+      name: "{{yourAPIKeyId}}",
+      options: {
+        permissionAllRead: false, // Does this API Key have read permission to all topics?
+        permissionAllWrite: false, // Does this API Key have write permission to all topics?
+      },
+    },
+  ];
+};
+```
```

Comment:
Oh! This template should have been updated as part of https://github.com/grouparoo/grouparoo/pull/2117!  I made a story for it, that's probably blocking this now - https://www.pivotaltracker.com/story/show/179111715

My fault!